docs/docs/faq/question_04.md

```diff
@@ -0,0 +1,29 @@
+
+# Can use the GPU for training the density estimator?
+
+TLDR; Yes, by passing `device="cuda"`. But no speed-ups for default density estimators.
+
+Yes. When creating the inference object in the flexible interface, you can pass the
+`device` as an argument, e.g.,
+
+```python
+inference = SNPE(simulator, prior, device="cuda:0", density_estimator="maf")
```

Comment:
is the `:0` needed? If not I'd prefer to take it out from the FAQ.

If it is needed, we should also add it in the TLDR